python/ray/data/_internal/logging.py

```diff
@@ -93,7 +99,12 @@ def configure_logging() -> None:
     environment variable. If the variable isn't set, this function loads the
     "logging.yaml" file that is adjacent to this module.
     """
-    config_path = os.environ.get("RAY_DATA_LOGGING_CONFIG", DEFAULT_CONFIG_PATH)
+    if "RAY_DATA_LOGGING_CONFIG" in os.environ:
```

Comment:
Does this UX make sense? Either supply the config file and use that or specify the mode correctly and use the corresponding default config file?

Reply:
IMO we should include "PATH" in the env var name. otherwise, this makes sense to me.